CONTRIBUTING.md

```diff
@@ -0,0 +1,59 @@
+# Contributing
+
+This file describes the process for contributing to `the-repo-name`.
+
+## Starting
+
+First and foremost, [fork](https://github.com/sedaprotocol/the-repo-name/fork) the repository. Then please read the
+[developing instructions](DEVELOPING.md) for setting up your environment.
+
+## Commits
+
+Your commits must follow specific guidelines.
+
+### Signed Commits
+
+Sign all commits with a GPG key. GitHub has extensive documentation on how to:
+
+- [Create](https://docs.github.com/en/authentication/managing-commit-signature-verification/generating-a-new-gpg-key)
+  a new GPG key.
+- [Add](https://docs.github.com/en/authentication/managing-commit-signature-verification/A-a-gpg-key-to-your-github-account)
+  a GPG key to your GitHub.
+- [Sign](https://docs.github.com/en/authentication/managing-commit-signature-verification/A-a-gpg-key-to-your-github-account)
+  your commits.
+
+### Convention
+
+All commits are to follow the
+[Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/) standard.
+Commit messages should always be meaningful.
+
+## Getting Ready For a PR
+
+This section describes actions to keep in mind while developing.
+
+### Change Size
+
+Please try to keep changes small to make reviews easier. We will reject more
+extensive unless there is a valid reason.
```

Comment:
Perhaps something like "Unless there is no way to break up the changes we will reject large PRs"